go.mod

```diff
@@ -4,6 +4,10 @@ go 1.16
 
 replace github.com/etcd-io/bbolt => go.etcd.io/bbolt v1.3.5
 
+// TODO: remove when the erigon-lib PR is merged
+replace github.com/ledgerwatch/erigon-lib => github.com/ledgerwatch/erigon-lib v0.0.0-20220308181950-238e94bb22d8
```

Comment:
It's ok to refer from Erigon's go.mod to erigon-lib's not merged PR (by commit hash).

Reply:
👌